lnwallet/btcwallet/log.go

```diff
@@ -0,0 +1,32 @@
+package btcwallet
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// Subsystem defines the logging code for this subsystem.
+const Subsystem = "BTWL"
```

Comment:
To avoid a circular package dependency I wasn't able to add it to the existing logger in `lnwallet/log.go`. So I decided to also give it its own subsystem name to distinguish it from the `btcsuite/btcwallet` package.